doc/lua_api.md

```diff
@@ -7845,6 +7845,27 @@ child will follow movement and rotation of that bone.
 * `get_bone_overrides()`: returns all bone overrides as table `{[bonename] = override, ...}`
 * `set_properties(object property table)`
 * `get_properties()`: returns a table of all object properties
+* `set_observers(observers)`: sets observers (players this object is sent to)
```

Comment:
there are two reasons why i want to be explicit here:

* there's more than visibility to "observability". there's sound for example; there may be an invisible object with sounds attached. (now that i think of it, i doubt that this is currently handled correctly; handling it correctly will probably have to be based on #14342)
* one specific reason this is interesting for modders is because it can be used to truly hide objects from players, even if cheat clients are involved.